neo4j/Dockerfile

```diff
@@ -1,5 +1,10 @@
 FROM neo4j:4.1.0
 
-COPY ./plugins /plugins
-RUN rm -r /var/lib/neo4j/data
-COPY ./data /var/lib/neo4j/data
+ARG NEO4J_USERNAME
+ARG NEO4J_PASSWORD
```

Comment:
Are these needed? From what I'm seeing they're defined as env vars instead.

Reply:
Continuing from above, providing the `--env-file` still does not make them available in the Dockerfile. Instead, one needs to defined the `ARGS`, which are only available during build (Dockerfile), see https://docs.docker.com/compose/compose-file/#args.